web-app/js/portal/cart/DownloadPanel.js

```diff
@@ -239,11 +239,6 @@ Portal.cart.DownloadPanel = Ext.extend(Ext.Panel, {
 
         params.onAccept = function(callbackParams) {
             self.downloader.download(collection, generateUrlCallbackScope, generateUrlCallback, callbackParams);
-            trackDownloadUsage(
-                OpenLayers.i18n('downloadTrackingActionPrefix') + OpenLayers.i18n(textKey),
-                collection.getTitle(),
-                undefined
-            );
         };
```

Comment:
What would be the indicator of which download option was chosen?
The url generated by the callback?